Xamarin.Forms.Platform.Android/AppCompat/CheckBoxRenderer.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.ComponentModel;
+using Android.Content;
+using Android.Support.V7.Widget;
+using Android.Widget;
+using Xamarin.Forms.Platform.Android.FastRenderers;
+
+namespace Xamarin.Forms.Platform.Android.AppCompat
+{
+	public class CheckBoxRenderer : ViewRenderer<CheckBox, AppCompatCheckBox>, CompoundButton.IOnCheckedChangeListener
```

Comment:
For new controls on android we just use the FastRenderer structure to eliminate the unnecessary ViewGroup in the hierarchy.  A slimmed down version of ButtonRenderer should work
https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/FastRenderers/ButtonRenderer.cs

Reply:
Also we just put them into the Android namespace for the new controls as well

https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/AppCompat/ImageButtonRenderer.cs#L15

Reply:
This is now addressed!